lib/hippo_file.rb

```diff
@@ -4,7 +4,7 @@ def filename
   end
 
   def blob
-    binary_blob = (find_property('hippo:text') || find_property('jcr:data')).value
+    binary_blob = (find_property('jcr:data') || find_property('hippo:text')).value
```

Comment:
There wasn't an original spec for this and I've not added one. I've only tested by actually running the script.